build/hydrobuild.md

```diff
@@ -0,0 +1,283 @@
+
+   The following command creates an `amd64` builder.
+   Replace `<org>` with the name of your Docker organization.
+
+   ```console
+   $ docker buildx create --driver cloud --name hydrobuild \
+     --platform linux/amd64 \
+     <org>/default_linux-amd64
+   ```
+
+   > **Note**
+   >
+   > A quick note on the positional argument for this command.
+   > This string represents the scope of the builder.
+   > It consists of three parts:
+   >
+   > - `<org>`: Docker organization.
+   > - `default`: Builder group. An arbitrary value that you can use to create
+   >   builders for different scopes in an organization. For example, if you
+   >   have multiple development teams, you can assign a unique builder to
+   >   each team using different builder groups.
+   > - `linux-amd64`: The build platform. Can be one of `linux-amd64` and
+   >   `linux-arm64`.
+
+3. If you also want to build for the `arm64` platform,
+   you can append another builder using the `--append` flag:
+
+   ```console
+   $ docker buildx create --append --name hydrobuild \
+     --platform linux/arm64 \
+     <org>/default_linux-arm64
+   ```
+
+## Using Hydrobuild
+
+To build your applications with Hydrobuild, you can:
+
+- [Use the Docker CLI](#use-hydrobuild-from-the-cli) to build from your local
+  development machine
+- [Use GitHub Actions](#use-hydrobuild-with-github-actions) to build with
+  Hydrobuild in CI
+
+### Use Hydrobuild from the CLI
+
+To run a build using Hydrobuild, invoke a build command and specify the
+name of the builder using the `--builder` flag.
+
+```bash
+docker build \
+  --builder hydrobuild \
+  --tag some-tag \
+  --load .
+```
+
+By default, the build result is only exported to the build cache. The `--load`
+flag ensures that the build output is loaded to your local image store.
+
+If you want to use Hydrobuild by default, you can run the following command to
+make it the selected builder:
+
+```console
+$ docker buildx use cloud-build --global
+```
+
+### Use Hydrobuild with GitHub Actions
+
+You can use GitHub Actions in combination with Hydrobuild to achieve faster
+build times, while still leveraging the convenience of GitHub Action workflows.
+
+With this approach, your CI workflows run on a GitHub Actions runner, and the
+runner calls out to the builder to build the image.
+
+To use Hydrobuild with GitHub Actions, you need to use the `lab` channel of
+`setup-buildx-action`:
+
+```yaml
+- name: Set up Docker Buildx
+  uses: docker/setup-buildx-action@v2
+  with:
+    version: "lab:latest"
+```
+
+The following example shows a basic workflow for GitHub Actions with Hydrobuild.
+
+```yaml
+name: ci
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          version: "lab:latest"
```

Comment:
@silvin-lubecki I think you mean `endpoint`:

```suggestion
          version: "lab:latest"
          driver: cloud
          endpoint: "<org>/<instance>"
```

Reply:
yes that 👍